api/v1alpha1/oidc_types.go

```diff
@@ -50,6 +51,35 @@ type OIDC struct {
 	// The path to log a user out, clearing their credential cookies.
 	// If not specified, uses a default logout path "/logout"
 	LogoutPath *string `json:"logoutPath,omitempty"`
+
+	// ForwardBearerToken indicates whether the Envoy should forward the access
+	// token as a bearer token in the "Authorization" header to the backend.
+	// If not specified, defaults to false.
+	// +optional
+	ForwardBearerToken *bool `json:"forwardBearerToken,omitempty"`
```

Comment:
This changes the current default behavior, which forwards the access token. 

We should mention this is the release doc.

Reply:
is there a batter name for `ForwardAccessTokenAsBearerToken` ?

Reply:
> is there a batter name for `ForwardAccessTokenAsBearerToken` ?

I think Bearer Token is normally an Access Token, so we probably don't need to emphasize that in the name?

Reply:
Thought about it again, it's better to make it clear even though the name becomes a bit longer.

Reply:
Change it to ForwardAccessToken and add a comment to explain that the access token will be forwarded via the Authorization header Bearer scheme to the upstream.